schema/resolvers.js

```diff
@@ -16,6 +17,15 @@ const assertMultipleChoiceAnswer = answer => {
   }
 };
 
+const getDefaultAnswerProperties = type => {
+  switch (type) {
+    case "Number":
+      return { required: true, decimals: 0 };
+    default:
+      return { required: true };
```

Comment:
How come we've switched the default case to mandatory/required being true instead of it being false like before?